modules/connected-to-disconnected-disconnect.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-disconnect_{context}"]
+= Disconnect the cluster from the network
+
+After you have mirrored all of the required repositories and configured your cluster to work as a disconnected cluster, you can then disconnect the cluster from the network. 
```

Comment:
If you trim a few words, does the meaning change too much? Maybe, "After mirroring all the required repositories and configuring your cluster to work as a disconnected cluster, you can disconnect the cluster from the network."